src/pages/productListing/components/filter/filter.js

```diff
@@ -27,81 +41,127 @@ const Filter = () => {
           <li className="mb-2 list-style-none">
             <input
               type="radio"
-              value="price-low-to-high"
+              value="LOW_TO_HIGH"
+              checked={state.sortBy === "LOW_TO_HIGH"}
               name="price"
               id="price-low-to-high"
+              onChange={(e) =>
+                dispatch({ type: "SORT", payload: { value: e.target.value } })
+              }
```

Comment:
you can put in single component to follow DRY

Reply:
yes I will try to do .